spec/en/reference.md

```diff
@@ -74,60 +76,87 @@ _Only files unique to GTFS-ride are defined below. Definitions for all other fil
+| ramp_used | Optional | The **ramp_used** field indicated whether a ramp or lift was used at the associated **stop_id**. This field should be used in conjunction with **ramp_boardings** and **ramp_alightings** if complete ramp/lift boarding and alighting counts are available. |
+|   |  | * **0** - No ramp/lift used.  |
+|   |  | * **1** - Ramp/lift deployed.  |
+| ramp_boardings | Optional | The **ramp_boardings** field contains the total count of ramp or lift deployed boardings at the identified **stop_id**. Non-negative integer. |
+| ramp_alightings | Optional | The **ramp_alightings** field contains the total count of ramp or lift deployed alightings at the identified **stop_id**. Non-negative integer. |
+| service_date | Optional | The **service_date** field contains the date of the associated boarding and/or alighting data at the identified stop. The format is YYYYMMDD. |
+| service_arrival_time | Optional | The **service_arrival_time** field contains the time of the actual arrival at the identified stop. The format is HH:MM:SS. |
+| service_departure_time | Optional | The **service_departure_time** field contains the time of the actual departure from the identified stop. The format is HH:MM:SS. |
 | source | Optional | The **source** field contains the collection method of the associated data. |
 |   |  | * **0** - Manual.  |
 |   |  | * **1** - APC.  |
 |   |  | * **2** - AFC.  |
 |   |  | * **3** - Model estimation.  |
 |   |  | * **4** - Mixed source.  |
 
+### *__trip_capacity.txt__*
+
+File: **Optional**
+
+The [trip_capacity.txt](#trip_capacitytxt) file identifies the capacity of a vehicle operating a trip on a given day to support crowding analysis in conjunction with [board_alight.txt](#board_alighttxt).
+
+|  Field Name | Required | Details |
+|  ------ | ------ | ------ |
+| agency_id | Optional | The **agency_id** field contains the ID of the agency associated with the capacity data. This value is referenced from the [agency.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#agencytxt) file. Use this field when you are providing data from more than one agency.  |
+| trip_id | Optional | The **trip_id** field contains an id that uniquely identifies a trip. If an agency has only one type of vehicle, or operated only one type of vehicle on the given **service_date**, it can leave **trip_id** blank to specify capacity of all trips with one record.    |
+| service_date | Optional | The **service_date** field contains the date of the trip. If an agency has only one type of vehicle, or only ever operates one type of vehicle on the given trip, it can leave **service_date** blank to specify capacity of all dates with one record. The format is YYYYMMDD. |
+| vehicle_description | Optional | The **vehicle_description** field contains the additional information about the vehicle associated with **trip_id** and **service_date** needed for analysis or reporting. |
```

Comment:
We can leave it as a string; do so will just challenge processing approach for any analytics.